azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/setup/components/ErrorInfoView.kt

```diff
@@ -76,9 +76,6 @@ internal class ErrorInfoView(private val rootView: View) {
         if (errorMessage.isBlank()) return
         snackBarTextView.text = errorMessage
         snackBar.run {
-            if (isShown) {
```

Comment:
Why was this removed?

Reply:
If snack bar is already shown, then it does close it, but a new one is not opened. 
From user perspective we do not need to close and open, the only what needs to change is a message.